packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -27,14 +29,25 @@ import {
     type PreferenceIds,
     type CookieBannerLocale,
     type CustomTagEnhancers,
+    DefaultProps,
 } from './defs';
 
 import { localiseText, localiseRichText } from './localisation-utils';
-import defaultLocale from '../locales/en-gb.json';
 
 // Valid values available to consumers
 export * from './defs';
 
+// TODO: Move this object to defs.ts
+// It can be moved as soon as we manage to figure a safe way to avoid the error TS2821
+// Import assertions are only supported when the '--module' option is set to 'esnext' or 'nodenext'
+export const defaultProps: Readonly<DefaultProps> = {
+    hasPrimaryActionsOnly: false,
+    defaultPreferences: {},
+    locale: defaultLocale,
+    cookieStatementLink: '',
+    cookieTechnologiesLink: '',
+};
```

Comment:
Browser tests for `pie-cookie-banner` fail when they find the import for the default JSON locale, **ONLY IF** it is in the `defs.ts` file. When the import is in the `index.ts` file the error doesn't happen. 😄

What is also peculiar is the error happening only in browser tests, never in compilation...

```
TypeError: Module "file:///.../pie/packages/components/pie-cookie-banner/locales/en-gb.json" needs an import assertion of type "json"
```

As suggested by the error message, a solution is to add `assert { type: 'json' };` to the import, however it will cause a TS error that requires to update our `tsconfig` `module` to `ESNext` (currently we use `ES2022`).

I'm not entirely sure if this is the reasonable or safe solution, so at the cost of consistency, I kept the JSON import in `index.ts`.

If we decide to go with `tsconfig` changes or some other settings update to browser tests, I'd suggest to add to follow-up PR so we avoid scope creep.